scripts/run_validator.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
-tag=latest
+tag=add-survey-form-type-region
```

Comment:
Revert validator branch.
```suggestion
tag=latest
```